### DISCUSSION TITLE
Considering using this. What should I be aware of in terms of format?

### DISCUSSION THREAD

**Discussion by philmetzger** [ggml-org/whisper.cpp#1579] [Q&A]
I am building a little app, where users can record their audio from microphone, upload it, and then get a transcription.

So my question is what format should I record the audio in? Are there any other settings I should be aware of?

**Comment by bobqianic** [comment]
> So my question is what format should I record the audio in? 

Any format.

> Are there any other settings I should be aware of?

The input audio needs to be in a mono PCM format with a 16KHz sample rate, saved as a WAV file. You can easily convert any audio file to this format using ffmpeg.
